### DISCUSSION TITLE
Client Connection pooling with Http/3

### DISCUSSION THREAD

**Discussion by hamnis** [netty/netty#13984] [Q&A]
Is there a way, or does it even make sense, to use connection pooling with quic channels?

In the example, it binds the bootstrap to a random port (0) and FixedChannelPool uses the client bootstrap to make connections.

This may be a clumsily worded question.

I currently have [this](https://github.com/http4s/http4s-netty/pull/636) in Scala.
For normal http/1.1 and http/2 i can use the standard FixedChannelPool just fine.

**Comment by chrisvest** [comment]
There's no socket associated with an individual QUIC stream, so you would just be pooling objects and dTLS state. It might be worth it for saving some TLS handshakes, but TLS session resumption might also make that point moot.

**Reply by He-Pin** [reply]
Yes, as the https://github.com/netty/netty-incubator-codec-http3/blob/main/src/test/java/io/netty/incubator/codec/http3/example/Http3ClientExample.java, I think the there is no need to do the pooling for quic channel.
